src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
nice, can you tell me more about the purpose of this metadata file?

Reply:
https://docs.spring.io/spring-boot/docs/current/reference/html/configuration-metadata.html
IntelliJ did this for me, I think we can remove them for now and add them in a more helpful condition later.